bytecomp/dll.mli

```diff
@@ -38,7 +38,13 @@ type dll_address
    Return [None] if the primitive is found in a DLL opened "for checking"
    (see above).
    Raise [Not_found] if not found. *)
-val find_primitive: string -> dll_address option
+
```

Comment:
Even more clear than my suggestion but you need to update and move the comment above to the docstring of `find_primitive`.

Reply:
Thanks, fixed!